test/unittest/tests/test_extent_list.cpp

```diff
@@ -0,0 +1,425 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <memory>
+
+#include <catch2/catch.hpp>
+
+#include "wt_internal.h"
+
+struct ExtentWrapper {
+    ExtentWrapper(WT_EXT *raw) : _raw(raw) {}
+
+    ~ExtentWrapper()
+    {
+        free(_raw);
+    }
+
+    WT_EXT *_raw;
+};
+
+struct ExtentListWrapper {
+    ExtentListWrapper() {}
+
+    ~ExtentListWrapper() {}
+
+    // unfortunately, some functions need raw 2D pointers, but that's not
```

Comment:
This comment uses `//` while the others use `/* */`. I think we should move to `//` for c++ but we need to be consistent. Same applies for the rest of the file.

Reply:
Thanks, fixed in 117f0bc. Regarding consistency, I like `//` for single-line comments and `/* */` for multiline comments - does that sound good to you?

Reply:
Sounds good to me, happy to have feedback from the others though. @keitharnoldsmith @jeremythorp

Reply:
In that case, my vote is that we stick with `//`. My biggest problem with `/* */` is that the lack of nesting makes it painful to put block quotes around sections of code.